code/modules/mob/living/carbon/monkey/inventory.dm

```diff
@@ -47,6 +47,7 @@
 	W.forceMove(src)
 	if(client)
 		client.screen |= W
+	INVOKE_EVENT(src, /event/equipped, W, slot)
```

Comment:
This event should be invoked by `equipped`, instead of being a responsibility of its callers.
There's another instance you can fix here: https://github.com/vgstation-coders/vgstation13/blob/d4a4ce82d76a71856f2f26d2a41512855b6e0db8/code/modules/mob/living/carbon/human/inventory.dm#L503